src/analytics/constants.ts

```diff
@@ -0,0 +1,138 @@
+/**
+ * Event names that can occur in this application.
+ *
+ * Subject to change as new features are added and new events are defined
+ * and logged.
+ */
+export enum EventName {
```

Comment:
Can we potentially fix the inconsistencies while we're here and have these enums all be the same casing convention? We have some that are CapitalCase and others CAPITAL_SNAKE_CASE. Is there a reason for the different conventions here?

If there is, it'd be good to document that and group accordingly!

Reply:
Same here for styling as above.

Reply:
resolved offline!